src/components/Articles/CompanyProvisions.vue

```diff
@@ -229,15 +223,22 @@ export default class CompanyProvisions extends Mixins(CommonMixin) {
   margin-left: 0;
 }
 
-::v-deep .v-input--selection-controls {
-  margin-top: 1.875rem;
-  padding: 0;
-}
-
-.checkbox-label {
+::v-deep .v-input--checkbox .theme--light.v-label {
   line-height: 1.375rem;
   font-size: 0.875rem;
   font-weight: normal;
+}
+
+::v-deep .invalid.v-input--checkbox .theme--light.v-label {
+  color: $app-red;
+}
+
+::v-deep #checkbox-div .v-input__slot {
+  align-items: start;
+  justify-content: start;
```

Comment:
Are you sure you need all this styling for the checkbox/label? I don't recall having to do this with other checkboxes. Please have another look (or explain the problem). Thx.

Reply:
The problem is keeping the checkbox to the top as UX required. 
![image](https://user-images.githubusercontent.com/78114138/110556580-acd54b80-80f3-11eb-9119-72eb707d74c3.png)

Reply:
OK, good explanation :)